exploration/number-selection.md

```diff
@@ -292,6 +367,12 @@ but can cause problems in target locales that the original developer is not cons
 When implementing `style=percent`, the numeric value of the operand
 MUST be divided by 100 for the purposes of formatting.
 
+> For example,
+> ```
+> .local $percent = {1000 :integer style=percent}
+> {{This formats as '10%' in the en-US locale: {$percent}}}
+> ```
```

Comment:
This is wrong, thankfully it's right in [the spec](https://github.com/unicode-org/message-format-wg/blob/main/spec/registry.md#percent-style).
```suggestion
When implementing `style=percent`, the numeric value of the operand
MUST be multiplied by 100 for the purposes of formatting.

> For example,
> ```
> .local $percent = {1 :integer style=percent}
> {{This formats as '100%' in the en-US locale: {$percent}}}
> ```
```